res/css/views/dialogs/_SpaceSettingsDialog.scss

```diff
@@ -78,7 +78,6 @@ limitations under the License.
     .mx_AccessibleButton_hasKind {
         &.mx_AccessibleButton_kind_link {
             font-size: $font-14px;
-            margin: 7px 18px;
```

Comment:
The style of `Upload` link button next to the avatar is specified with `_SpaceBasicSettings.scss` so removing this line does not change that style.